libraries/provider_hab_service_systemd.rb

```diff
@@ -0,0 +1,79 @@
+# Copyright:: Copyright 2016, Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "chef/provider/service"
+
+class Chef
+  class Provider
+    class Service
+      class HabServiceSystemd < Chef::Provider::Service
+
+        use_inline_resources
+
+        provides :hab_service_systemd
+        provides :hab_service do |node|
+          Chef::Platform::ServiceHelpers.service_resource_providers.include?(:systemd)
+        end
+
+        action :start do
```

Comment:
I wasn't intending on it for this MVP. What would that look like? Aren't all systemd units that end in`.service` enabled by default? I don't know how to systemd, but that was my impression.